cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,76 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import SettingsPageObject from '../support/pages/settings.PageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-describe('Settings page', () => {
-  before(() => {
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const signInPage = new SignInPageObject();
 
-  });
+describe('Settings page', () => {
+  let newData;
+  let user;
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.task('generateNewData').then(generateNewData => {
+      newData = generateNewData;
+    });
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+    cy.register();
```

Comment:
It's not clear what data `cy.register()` is using for registration. Consider passing `user` object as an argument to make sure that the correct data is used for registration.